src/cocaine-app/planner/__init__.py

```diff
@@ -1410,6 +1491,7 @@ def ttl_cleanup(self, request):
 
         iter_group = storage.groups[request['iter_group']]
 
+        logger.fatal("Create ttl_cleanup_job for couple {!s}".format(iter_group.couple))
```

Comment:
Done in https://github.com/yandex/mastermind/pull/311/commits/4732106cd00d7b48fb9a6db30b7cff4aef8db6df